lib/spack/docs/packaging_guide.rst

```diff
@@ -2007,10 +2007,15 @@ The last element of a package is its ``install()`` method.  This is
 where the real work of installation happens, and it's the main part of
 the package you'll need to customize for each piece of software.
 
-.. literalinclude::  ../../../var/spack/repos/builtin/packages/mpfr/package.py
-   :pyobject: Mpfr.install
+.. code-block:: python
    :linenos:
 
+   def install(self, spec prefix):
+       configure('--prefix={0}'.format(prefix))
+
+       make()
+       make('install')
```

Comment:
This will just be overwritten by #2780, but I needed to do something to get the doc tests to pass.